website/docs/uses.md

```diff
@@ -0,0 +1,23 @@
+Specifies which usage blocks a project should consume.
+
+```lua
+uses { "ProjectA" }
+```
+
+The `uses` API is used to consume `usage` blocks from within a project. The `usage` blocks are case sensitive.
+
+### Usage resolve priority ###
+
+1. `PUBLIC` and `INTERFACE` usage scopes within a project of the corresponding name.
+2. `usage` blocks with the corresponding name in any scopes.
+
+Note: If there are duplicate usage blocks with the same resolved name, the selected usage block is unspecified. `usage` blocks should have unique names if they are not specified as `PUBLIC`, `PRIVATE`, or `INTERFACE`. If `links` or `linkoptions` are defined within a usage block, it is recommended that `linkgroups` is also turned on.
```

Comment:
Duplicate usage blocks: This should error in Premake, or "reopen" the usage block.

`linkgroups`: I don't think I really agree with re-documenting this. Will every API that ends up having edge cases be re-documented here? `uses` doesn't create any issues with `links`, if anything, it resolves them by having projects state their dependencies which can alleviate the ordering issues.

Reply:
Duplicate usage blocks (custom names) will be addressed in a follow on PR.